nailgun/entities.py

```diff
@@ -21,8 +21,12 @@
 workings of entity classes.
 
 """
+# pylint:disable=no-member
```

Comment:
removed.. (it was added because locally I was using updated version of Pylint and 90% of the file fails with this...